kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/types/listview/DataTypeListItemView.java

```diff
@@ -377,6 +377,12 @@ public void hideDataTypeNameInput() {
     @Override
     public void setupSelectComponent(final DataTypeSelect typeSelect) {
         type.innerHTML = "";
+
+        HTMLElement element = typeSelect.getElement();
```

Comment:
I think we could add a final here:
```suggestion
        final HTMLElement element = typeSelect.getElement();
```